### ISSUE TITLE
Unable to comment code inside styled-components

### ISSUE THREAD

**Issue by zuha-khalid-au3** [styled-components/styled-components#3403]
I am trying to add comment to one of css inside styled-comments but it's throwing Component Exception on both Android and iOS

Example1- 
const SearchContainer = styled.View`
//commenting here
padding:14px;
`;

But when I am trying to add comment on second line onwards then it's considering it as comment that too only in iOS

Example2- 
const SearchContainer = styled.View`
flex:1;
//commenting here
padding:14px;
`;

In short I mean to say that Example 2 is working fine with iOS but not with Android.

**Comment by kitten** [comment]
Please refer with usage questions to Stack Overflow, as you're more likely to get quicker and helpful answers there ❤️ Repository issues on GitHub are great for that, issues, things that need to be resolved in the library itself, or can't be resolved by anyone but the maintainers. While I (and many others on other repos) are happy to eventually answer usage questions, styled-components is a busy repo and mostly stable and slow moving towards new releases, so you're not likely to get fast responses here
